<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CommandReactionConfigDialog</class>
 <widget class="QDialog" name="CommandReactionConfigDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>712</width>
    <height>494</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit command reaction</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>reaction</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QRadioButton" name="reactOkBtn">
           <property name="text">
            <string>ok</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="reactErrBtn">
           <property name="text">
            <string>err</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="reactManualBtn">
           <property name="text">
            <string>manual</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="reactJsBtn">
           <property name="text">
            <string>exec javascript code</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="reactStuckBtn">
           <property name="text">
            <string>set not working</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="reactResetBtn">
           <property name="text">
            <string>reset</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="reactIgnoreBtn">
           <property name="text">
            <string>ignore</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="editStateChangeBeforeAnswerBtn">
           <property name="text">
            <string>edit state changes
before answer</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="editStateChangeAfterAnswerBtn">
           <property name="text">
            <string>edit state changes
after answer</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QStackedWidget" name="stackedWidget">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="answerWidget">
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QWidget" name="widget_2" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QPushButton" name="addAnswArgBtn">
               <property name="text">
                <string>add arg</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="delAnswArgBtn">
               <property name="text">
                <string>del arg</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>287</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QListWidget" name="argsList"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="jsEditWidget">
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>javascript code</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTextEdit" name="jsCodeEdit"/>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CommandReactionConfigDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CommandReactionConfigDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
